src/test/java/org/jabref/logic/plaintextparser/ParserPipelineTest.java

```diff
@@ -0,0 +1,65 @@
+package org.jabref.logic.plaintextparser;
+
+import org.junit.Test;
+
+/**
+ * Used to test the whole logic of the external parser anystyle.
+ * Therefore any possible outgoing when the user is working with the parser is going to be
+ * tested in this class.
+ */
+
+public class ParserPipelineTest {
+
+    /**
+     * Tests the base functionality of the parser is working by taking some example
+     * strings and parses them with the external parser and checks if the corresponding fields
+     * are filled in correctly.
+     */
+    @Test
+    public void singleTextResourceParseTest() {
+      ParserPipeline.parsePlainRefCit("Derwing, T. M., Rossiter, M. J., & Munro, M. J. (2002). Teaching native speakers to listen to foreign-accented speech. Journal of Multilingual and Multicultural Development, 23(4), 245-259.");
```

Comment:
Please add an Assertion also during development. At the beginning `assertNotNull` might be enough (so that you don't need to adapt the tests all the time)